README.md

```diff
@@ -127,4 +127,31 @@ Follow the instructions to login to GitHub using your credentials.
 Follow the instructions specified, then run this command again:<br/>
 `git push origin {your-branch-name}`
 
+## Testing of Sample Queries
+Once you've added/updated the sample queries, you can test them out by going to [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer), and appending this to the Graph Explorer url.
+
+`?devx-api=https://graphexplorerapi.azurewebsites.net&org=microsoftgraph&branchName=dev`
```

Comment:
The `org` and `branch` needs to be specific to where the samples were added in. Could be this repo in a diff. branch or a forked repo of this repo.